examples/load_ui_base_instance.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import os
+
+os.environ["QT_PREFERRED_BINDING"] = "PySide"
```

Comment:
Ok. If unspecified, it will try and use PySide2. Maybe there's some issue there.